cocotb/share/makefiles/Makefile.sim

```diff
@@ -136,11 +136,7 @@ define check_for_results_file
     @test -f $(COCOTB_RESULTS_FILE) || (echo "ERROR: $(COCOTB_RESULTS_FILE) was not written by the simulation!" >&2 && exit 1)
 endef
 
-# Append directory containing the cocotb Python package to PYTHONPATH
-# XXX: This is only needed if cocotb is not installed into the default
-# Python search path.
-PYTHONPATH := $(COCOTB_PY_DIR):$(PYTHONPATH)
-export PYTHONPATH
+PYTHONPATH := $(PYTHONPATH):$(PWD)
```

Comment:
This is backward, `PWD` is always inserted first on the path by a regular python interpreter
```suggestion
PYTHONPATH := $(PWD):$(PYTHONPATH)
```

Reply:
This will fail because of `:` in case `PYTHONPATH` is empty (heh) so need to guard this with `ifdef` or add to `sys.path` inside cocotb/embedded interpreter.

Reply:
Shouldn’t the path separator be ‘;’ rather than ‘ :’ on windows?

Reply:
Hmm, good catch. I think the msys path-conversion stuff doesn't kick in unless the colon has a character after it.

Reply:
No. msys understand `:`.

Reply:
(edited my comment above)

Reply:
@eric-wieser 
```
['C:\\Users\\Tomek\\git\\cocotb\\examples\\adder\\model',
 'C:\\Users\\Tomek\\git\\cocotb\\examples\\adder\\tests',
 'C:\\Users\\Tomek\\Miniconda3\\python37.zip',
 'C:\\Users\\Tomek\\Miniconda3\\DLLs',
 'C:\\Users\\Tomek\\Miniconda3\\lib',
 'C:\\Users\\Tomek\\iverilog\\bin',
 'C:\\Users\\Tomek\\Miniconda3',
 'C:\\Users\\Tomek\\Miniconda3\\lib\\site-packages',
 'c:\\users\\tomek\\git\\cocotb-coverage',
 'c:\\users\\tomek\\git\\cocotb-test',
 'c:\\users\\tomek\\git\\cocotb',
 'C:\\Users\\Tomek\\Miniconda3\\lib\\site-packages\\win32',
 'C:\\Users\\Tomek\\Miniconda3\\lib\\site-packages\\win32\\lib',
 'C:\\Users\\Tomek\\Miniconda3\\lib\\site-packages\\Pythonwin']
```

Reply:
Discussed out of band - this issue is minor, and @themperek have a solution to eliminate `PYTHONPATH` entirely. Let's go ahead and merge.